src/components/Avatar/index.tsx

```diff
@@ -27,13 +29,16 @@ const StyledDiv = styled.div<{
   textSize: number
 }>`
   align-items: center;
-  background-color: ${({ lock, theme, textBgColor }) =>
-    lock
-      ? theme.colorsDeprecated.gray50
-      : theme.colorsDeprecated[textBgColor as Color] || textBgColor};
+  background-color: ${({ lock, theme, textBgColor }) => {
+    if (lock) return theme.colors.neutral.background
+    const sentimentColor =
+      theme.colors[textBgColor as Sentiment]?.backgroundStrong
+
+    return sentimentColor ? transparentize(0.5, sentimentColor) : textBgColor
```

Comment:
Why transparentize here ?

Reply:
Transparentize should disappear with usage of secondary (see my comment below)

Reply:
done